content/docs/ops/runbook/troubleshooting-kubernetes.md

```diff
@@ -14,26 +14,39 @@ Alerts are generated whenever a pod's status is not `Running`. Alerts contains t
 
 If the namespace is `default` and the pod name is a random string then it is a managed service and likely only impacts a specific tenant application.  If the namespace is `kube-system` or the pod name is human readable then it is a platform component which could impact all managed services.
 
-#### Fixing non-running pods
-Login to a kubernetes master and describe the pod:
+#### Login to kubernetes master
+
+From a jumpbox in the appropriate environment:
+- ssh to a master
+- update the PATH to find the kubernetes binaries
+- sudo to root for all the `kubectl` commands
+
 ```sh
-/var/vcap/packages/kubernetes/bin/kubectl --namespace :namespace describe pod :pod-name
+bosh -d kubernetes ssh master/0
+PATH=$PATH:/var/vcap/packages/kubernetes/bin
+sudo bash
```

Comment:
When I reviewed this I'd thought I'd gotten them out of order too.... but it worked and I just triplechecked:

![screen shot 2017-08-11 at 12 25 04 pm](https://user-images.githubusercontent.com/1211146/29222386-4598861c-7e90-11e7-8fc1-49df6b6b5c7b.png)

However, I'll stick to the "principle of least surprise" reorder those.